### PR TITLE
fuzzer: Update to EVMC7, use Aleth 1.7.0-rc.1

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: test/fuzzer/CMakeLists.txt [hunk 1]
```diff
@@ -21,7 +21,7 @@ if(NOT fuzzing_coverage)
     # TODO: Aleth reports undefined behaviors, disable it for fuzzing.
     string(REPLACE undefined "" aleth_fuzzing_flags ${fuzzing_flags})
 
-    set(aleth_git_tag v1.7.0-alpha.1)
+    set(aleth_git_tag v1.7.0-rc.1)
```
Comment (axic) [inline test/fuzzer/CMakeLists.txt]: Shouldn't this be matched by bumpversion too? If not, should also fix the bumpversion config.

**Comment by axic** [comment]
Where is this fuzzer run?

@bshastry would it be possible to submit this to ossfuzz?

File: test/fuzzer/CMakeLists.txt (hunk 1 above)
Comment (axic) [reply test/fuzzer/CMakeLists.txt]: Ah never mind, this is the fixed version it loads.

**Comment by chfast** [comment]
> Where is this fuzzer run?
> 
> @bshastry would it be possible to submit this to ossfuzz?

So far it runs on my machine. I need to put some work into Aleth's CMake to allow building aleth-interpreter without all Aleth's dependencies.

File: test/fuzzer/fuzzer.cpp [hunk 2]
```diff
@@ -230,7 +232,7 @@ fuzz_input populate_input(const uint8_t* data, size_t data_size) noexcept
     if (data_size < input_size_16bits)  // Not enough data for input.
         return in;
 
-    in.rev = rev_4bits > EVMC_PETERSBURG ? EVMC_PETERSBURG : static_cast<evmc_revision>(rev_4bits);
+    in.rev = rev_4bits > latest_rev ? latest_rev : static_cast<evmc_revision>(rev_4bits);
```
Comment (axic) [inline test/fuzzer/fuzzer.cpp]: Why not fuzz for older versions though?

Comment (chfast) [reply test/fuzzer/fuzzer.cpp]: It does when `rev_4bits < latest_rev`. Only the `latest_rev` happens more frequently.

**Comment by bshastry** [comment]
> Where is this fuzzer run?
> 
> @bshastry would it be possible to submit this to ossfuzz?

I guess I'm missing context. What is being fuzzed here? In general it should be possible to upstream it to ossfuzz if a static binary can be built because the machines on which google fuzzes a binary are different from where they are built.

**Pull request merged**